l10n_th_vat_report/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_report_vat_report,access_report_vat_report,model_report_vat_report,,1,0,0,0
```

Comment:
Normally, models.TransientModel do not require access right.
So, I think you can remove this file (please double that it still works).